### ISSUE TITLE
We will remove the plugin Chat from the marketplace on request of Vincent

### ISSUE THREAD

**Issue by mattab** [VincentLahaye/piwik-chat#25]
Hello everyone,

Vincent requested to remove the plugin from the marketplace as it appears to have bugs.

Would someone be willing to "take over" / maintain the plugin in the future?

If nobody steps up soon, we will remove the plugin from the Marketplace, on request of the author. 

Thank you!

**Comment by mattab** [comment]
cc @VincentLahaye

**Comment by mattab** [comment]
Actually maybe we could just leave it on the Marketplace anyway, as soon as we will only show by default the "Piwik 3.X" compatible plugins, so the Chat plugin would be hidden by default unless a user selects "Piwik 2.X" in the selector. @tsteur could you confirm that we will soon, only show Piwik 3 compatible plugins by default on the homepage of marketplace?

**Comment by tsteur** [comment]
no, we will show all plugins for all piwik versions on homepage. Only when filter is applied we will filter
